db/migrate/20210407104300_downcase_mac_addresses.rb

```diff
@@ -0,0 +1,8 @@
+class DowncaseMacAddresses < ActiveRecord::Migration[6.0]
+  def up
+    Nic::Managed.where.not(mac: nil).find_each { |nic| nic.update mac: nic.mac.downcase }
```

Comment:
Isn't there a pure SQL way to do this? Untested, but perhaps this works?
```suggestion
    Nic::Managed.where.not(mac: nil).update_all('mac = lower(mac)')
```

Reply:
Even better: `execute 'UPDATE nics SET mac=LOWER(mac)'`